dashboard/app/helpers/levels_helper.rb

```diff
@@ -499,6 +499,10 @@ def blockly_options
       app_options['pusherApplicationKey'] = CDO.pusher_application_key
     end
 
+    if @game == Game.applab
+      app_options['safeBrowsingKey'] = CDO.google_safe_browsing_key
```

Comment:
I don't love this solution of just adding the key to appOptions. Any ideas on how we could still surface this key in JS, but not through AppOptions?

Reply:
For pusher, we use an App ID but it's not a secret, and it doesn't get abused (probably) because it also depends on a server-side API where we use a secret.  WebPurify is entirely handled on the server-side.  I'd be tempted to do that here as well - at the very least, proxy through our server, for two reasons: 1) to hide this key, and 2) so that schools with restricted firewalls only need to worry about allowlisting our domains.

Reply:
+1, this should be done on our server so that the key is kept private.